Gulpfile.mjs

```diff
@@ -292,6 +292,10 @@ function buildRollup(packages, targetBrowsers) {
               "packages/babel-compat-data/*.js",
               "packages/*/src/**/*.cjs",
             ],
+            dynamicRequireTargets: [
+              // https://github.com/mathiasbynens/regexpu-core/blob/ffd8fff2e31f4597f6fdfee75d5ac1c5c8111ec3/rewrite-pattern.js#L48
+              "node_modules/regenerate-unicode-properties/**",
```

Comment:
It's not guaranteed that it's always inside the top-level `node_modules`, since it's a nested dependency.

WDYT about creating a utility like this?
```js
import { createRequire } from "module";
import { fileURLToPath } from "url";
import { dirname } from "path";

/**
 * Resolve a nested dependency starting from the given file
 */
export default function resolveChain(baseUrl, ...packages) {
  const require = createRequire(baseUrl);

  return packages.reduce(
    (base, pkg) => require.resolve(pkg, { paths: [dirname(base)] }),
    fileURLToPath(baseUrl)
  );
}
```

And then we can use
```js
resolveChain(
  import.meta.url,
  "packages/babel-helper-create-regexp-features-plugin",
  "regexpu-core",
  "regenerate-unicode-properties"
);
```

Also, maybe we can end the pattern with `**/*.js` to only include JS files?

Reply:
Thanks! That looks fancy! I think it is okay to use `**/*`. If we have `require("foo/bar.md")`, rollup will throw when bundling `bar.md`.

Reply:
We can use `regenerate-unicode-properties/package.json` as the last param to avoid that breakage.